app/client/src/pages/Editor/Explorer/Entity/EntityProperties.tsx

```diff
@@ -62,7 +62,8 @@ export const EntityProperties = memo(
                 actionProperty = actionProperty + "()";
               }
               if (actionProperty === "data") {
-                value = entity.data?.body;
+                value =
+                  entity.data?.body === undefined ? "{}" : entity.data?.body;
```

Comment:
The type protection is not done correctly over here.
Instead try doing something like this.
```suggestion
                if(!entity.data.body) {
                  value = undefined
                } else {
                  value = entity.data.body
                }
```

Reply:
I did the changes for adding type protection, can you please re-check?